packages/contracts-bedrock/foundry.toml

```diff
@@ -25,8 +25,6 @@ additional_compiler_profiles = [
 compilation_restrictions = [
   { paths = "src/dispute/FaultDisputeGame.sol", optimizer_runs = 5000 },
   { paths = "src/dispute/PermissionedDisputeGame.sol", optimizer_runs = 5000 },
-  { paths = "src/dispute/SuperFaultDisputeGame.sol", optimizer_runs = 5000 },
-  { paths = "src/dispute/SuperPermissionedDisputeGame.sol", optimizer_runs = 5000 },
```

Comment:
why are these removed?

Reply:
These contracts are below the bytecode limit without needing to change their optimizer settings, we prefer to have the default compiler settings if possible